src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.c

```diff
@@ -358,7 +357,7 @@ static grpc_ares_request *grpc_dns_lookup_ares_impl(
     grpc_ares_request_ref(r);
     char *service_name;
     gpr_asprintf(&service_name, "_grpclb._tcp.%s", host);
-    ares_query(*channel, service_name, ns_c_in, ns_t_srv, on_srv_query_done_cb,
+    ares_query(*channel, service_name, 1 /*ns_c_in*/, 33 /*ns_t_srv*/, on_srv_query_done_cb,
```

Comment:
@y-zeng They should not be OS dependent - [`ares_query`](https://linux.die.net/man/3/ares_query) is a DNS request:
- `dnsclass`: `ns_c_in == 1` means "internet" ([RFC 1035, section *3.2.4. CLASS values*](https://www.ietf.org/rfc/rfc1035.txt)) and
- `type`: `ns_t_srv == 33` means "server selection" ([SRV Record](https://en.wikipedia.org/wiki/SRV_record))

Shall I add a simple comment in the line above or the whole `nameser.h` file? (it's not really "from c-ares" but all-over the web in all kinds of DNS based libs and apps; can't track the origin)